src/Adapter/CMS/PageCategory/CommandHandler/BulkDeleteCmsPageCategoryHandler.php

```diff
@@ -44,7 +44,7 @@ final class BulkDeleteCmsPageCategoryHandler implements BulkDeleteCmsPageCategor
      *
      * @throws CmsPageCategoryException
      */
-    public function handle(BulkDeleteCmsPageCategoryCommand $command)
+    public function handle(BulkDeleteCmsPageCommand $command)
```

Comment:
Looks like a mistake 🤔

Reply:
Nice catch! Inded there were - the commands are so similar that I confused names here :open_mouth: .  I guess integration tests will help to defend from these kind of errors - after the freeze we should start to do that I believe :+1: